manage_breast_screening/clinics/jinja2/clinics/show.jinja

```diff
@@ -70,7 +70,7 @@
         appointment_status(presented_appointment)
       }}
       {% if presented_appointment.status_attribution %}
-        <span class="app-text-grey app-nowrap nhsuk-body-s nhsuk-u-margin-bottom-0">{{ presented_appointment.status_attribution }}</span>
+        <span class="nhsuk-hint">{{ presented_appointment.status_attribution }} {{presented_appointment.attribution_user_check(request.user)}}</span>
```

Comment:
Should this be a paragraph to prevent any strange wrapping?

```suggestion
        <p class="nhsuk-u-secondary-text-colour">{{ presented_appointment.status_attribution }} {{presented_appointment.attribution_user_check(request.user)}}</p>
```